pkg/api/v1/types.go

```diff
@@ -280,8 +280,25 @@ type PersistentVolumeSpec struct {
 	// ClaimRef is expected to be non-nil when bound.
 	// claim.VolumeName is the authoritative bind between PV and PVC.
 	ClaimRef *ObjectReference `json:"claimRef,omitempty" description:"when bound, a reference to the bound claim"`
+	// PersistentVolumeReclaimPolicy represents what happens to a persistent volume when released from its claim.
+	PersistentVolumeReclaimPolicy PersistentVolumeReclaimPolicy `json:"reclamationPolicy,omitempty" description:"reclamationPolicy is what happens to a volume when released from its claim; one of Recycle, Delete, Retain.  Default is Retain."`
```

Comment:
Please keep the type and struct field names similar to the json field name: PersistentVolumeReclamationPolicy.